test/test_evaluation/test_train_evaluator.py

```diff
@@ -393,7 +393,7 @@ def test_partial_cv(self, pipeline_mock):
         self.assertRaises(queue.Empty, evaluator.queue.get, timeout=1)
 
         self.assertEqual(evaluator.file_output.call_count, 0)
-        self.assertEqual(rval['loss'], 0.46666666666666667)
+        self.assertEqual(rval['loss'], 0.5)
```

Comment:
Do you have any idea what happened to these few test for cross-validation?

Reply:
TL;DR: @mfeurer , you were right, sklearn 21.3 and 22.1 generate different splits for the same given seed.

More details (level 1): Not only the chosen samples differ, but also the number of samples. For instance, this line that you highlighted: both the development branch and this PR's branch get 8 itens of the test set correctly classified. However, the split generated by sklearn 21.3 has 15 samples, an the one generated by 22.1 has 16, therefore the losses are respectively 0.46667 and 0.5.

More details (level 2): In order to double check this hypothesis, I performed one more sanity check: 
* run the test using the development branch and sklearn 22.1 and
* run the test using this PR's branch with sklearn 21.3. 
(In other words, I tried the old sklearn with the new branch and the new sklearn with the old branch.)
The accuracy I got for the 1st experiment was .46667 and for the 2nd it was .5. This supports the hypothesis that the cause of this discrepancy has nothing to do with auto-sklearn's code.

Reply:
Okay, thanks a lot for checking this.